try

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 luarocks remove busted --force
-luarocks make
+luarocks make --pin
```

Comment:
Depending on whether the Lua interpretter can be unpinned, using pinning in these two utility scripts probably doesn't make sense.

Reply:
While I have reverted the `--pin` for utility scripts, but I still strongly think they should have it. It would help with the bug reporting as well.